common/pom.xml

```diff
@@ -38,6 +38,11 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-smallrye-metrics</artifactId>
```

Comment:
@jsenko perhaps this brings in quarkus-core  dep ... ?

Reply:
Needs to be `provided`?